src/main/java/com/google/api/gax/grpc/ApiCallable.java

```diff
@@ -159,8 +160,19 @@ public ResponseT call(RequestT request) {
    */
   public ApiCallable<RequestT, ResponseT> retrying(
       RetryParams retryParams, ScheduledExecutorService executor) {
+    return retrying(retryParams, executor, DefaultClock.getInstance());
+  }
+
+  /**
+   * Creates a callable which retries using exponential back-off. Back-off parameters are defined
+   * by the given {@code retryParams}. Clock provides a time source used for calculating
+   * retry timeouts.
+   */
+  @VisibleForTesting
+  protected ApiCallable<RequestT, ResponseT> retrying(
```

Comment:
You could use package-private here (i.e. remove the "protected" and have no access modifier)